example/main.go

```diff
@@ -56,6 +57,7 @@ func main() {
 	http.HandleFunc("/profile", profile)
 	http.Handle("/images/", http.StripPrefix("/images",
 		http.FileServer(http.Dir(path.Join(rootdir, "images/")))))
+	log.Printf("About to listen and serve on 8080. Go to http://your-local-url:8080/")
```

Comment:
Can we have `localhost` or `127.0.0.1` instead of `your-local-url`? I know the docs are being more generic, but I'd rather be more explicit in the example. People can change the code anyway. Once we have this, this is good to merge.